boards/ikea-tradfri/doc.txt

```diff
@@ -2,4 +2,169 @@
 @defgroup    boards_ikea-tradfri IKEA TRÅDFRI modules
 @ingroup     boards
 @brief       Support for the IKEA TRÅDFRI modules
+
+## Overview
+The IKEA TRÅDFRI is a small board found in different IKEA TRÅDFRI products.
+The board contains an EFR32 microcontroller and a 2MBit SPI NOR Flash.
+
+More information about the module can be found on this
+[GitHub page](https://github.com/basilfx/TRADFRI-Hacking).
+
+## Hardware
+
+### MCU
+| MCU             | EFR32MG1P132F256GM32                                                                         |
+|-----------------|----------------------------------------------------------------------------------------------|
+| Family          | ARM Cortex-M4F                                                                               |
+| Vendor          | Silicon Labs                                                                                 |
+| Vendor Family   | EFR32 Mighty Gecko 1P                                                                        |
+| RAM             | 31.0KB                                                                                       |
+| Flash           | 256.0KB                                                                                      |
+| EEPROM          | no                                                                                           |
+| Frequency       | up to 38.4 MHz                                                                               |
+| FPU             | yes                                                                                          |
+| MPU             | yes                                                                                          |
+| DMA             | 12 channels                                                                                  |
+| Timers          | 2x 16-bit + 1x 16-bit (low power)                                                            |
+| ADCs            | 12-bit ADC                                                                                   |
+| UARTs           | 3x UART, 2x USART, 1x LEUART                                                                 |
```

Comment:
```suggestion
| UARTs           | 3x UART: 2x USART, 1x LEUART                                                                 |
```

Reply:
Fixed. I accidentally copied this information wrong. It only has 2x USART.